app/assets/javascripts/shopify_app/storage_access.js

```diff
@@ -1,3 +1,4 @@
+//= require ./app_bridge_redirect.js
```

Comment:
Do we need this?

Reply:
Yes, this is the shared helper used to call App Bridge to perform the redirect